torch/csrc/jit/codegen/cuda/utils.cpp

```diff
@@ -110,7 +110,8 @@ auto parseDisableOptions() {
       {DisableOption::IndexHoist, false},
       {DisableOption::Nvtx, false},
       {DisableOption::PredicateElimination, false},
-      {DisableOption::UnrollWithRng, false}};
+      {DisableOption::UnrollWithRng, false},
+      {DisableOption::ArchCheck, false}};
```

Comment:
I tried to order these options alphabetically, which I think would be helpful to find options when the option set becomes larger.